src/azure-cli/service_name.json

```diff
@@ -548,5 +548,10 @@
     "Command": "az restore-point",
     "AzureServiceName": "Virtual Machines",
     "URL": "https://docs.microsoft.com/rest/api/compute/restore-points"
+  },
+  {
```

Comment:
Please add after container, same as doc_source_map.json.
And I would really appreciate it if you could help move the restore-point after `az rest` as well.